### ISSUE TITLE
Compatibility with babel@7.5.0: Don't break the default exported statement

### ISSUE THREAD

**Issue by qiqiboy** [gaearon/react-hot-loader#1292]
### Description

RHL makes the  'export default' statement omitted while using @babel/plugin-transform-typescript@7.5.1.

### Expected behavior

Keep the 'export default' statement.

### Environment
```json
{
  "react-ho-loader": "^4.12.3",
  "@babel/core": "^7.5.0",
  "@babel/cli": "^7.5.0",
  "@babel/plugin-transform-typescript": "^7.5.1"
}
```

### Reproducible Demo

Input:
```typescript
export default 'foo';
```

Output:
```javascript
(function () {
  var enterModule = (typeof reactHotLoaderGlobal !== 'undefined' ? reactHotLoaderGlobal : require('react-hot-loader')).enterModule;
  enterModule && enterModule(module);
})();

var __signature__ = typeof reactHotLoaderGlobal !== 'undefined' ? reactHotLoaderGlobal.default.signature : function (a) {
  return a;
};

const _default = 'Foo';
; // 'export default _defualt'' was Omitted <<<<<<<<<<<<<<<

(function () {
  var reactHotLoader = (typeof reactHotLoaderGlobal !== 'undefined' ? reactHotLoaderGlobal : require('react-hot-loader')).default;

  if (!reactHotLoader) {
    return;
  }

  reactHotLoader.register(_default, "default", "/Users/qiqiboy/develop/tigerNewTest/babel-debug/test.tsx");
})();

;

(function () {
  var leaveModule = (typeof reactHotLoaderGlobal !== 'undefined' ? reactHotLoaderGlobal : require('react-hot-loader')).leaveModule;
  leaveModule && leaveModule(module);
})();
```

### The related issue

https://github.com/babel/babel/issues/10162#issuecomment-508985150

> https://github.com/gaearon/react-hot-loader/blob/master/src/babel.dev.js#L118
insertBefore returns an array of ast paths, and that plugin should call path.scope.registerDeclaration(resultOfInsertBefore[0])

Maybe I should open a PR, but I don't know how to do it.

**Comment by theKashey** [comment]
Hooray! Break the web again!
PS: PR drafted

**Comment by theKashey** [comment]
Released in 4.12.5
